### PR TITLE
Include option to install using npm ci

### LINKED ISSUE #2: Option to use `npm ci`
npm v5.7 introduced `npm ci` - https://docs.npmjs.com/cli/ci.html

I reckon this will be a good fit for this project.

Reason:
1. When someone upgrades/downgrades/removes/adds a pkg it will update the lock file.
2. npm ci will not update any files (ie the lock file)
3. Keeping the state as declared by the previous commit that actually did the package.json update.
4. It'll be quicker to install
5. albeit rm-s node_modules and reinstall, this will give you a good state as expected by the commit that did the package.json change.

### PR DESCRIPTION
**Pull request by jonyeezs**
Resolves #2 

Provides an option to use `npm ci` as the installation command.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -47,6 +71,15 @@ const exec = (cwd, opts) => new Promise((resolve, reject) => {
 	})
 });
 
+const checkNpmCiExist = () => exec('npm ci --help')
+.then(output => output.toString() === 'npm ci');
+
+const doesPackageJsonExist = () => new Promise((resolve) => {
```
Comment (jonathantneal) [inline src/index.js]: Would `doesPackageLockJsonExist` better describe what this is doing?

**Comment by jonyeezs** [comment]
Current implementation still checks on package.json file as what npm install would do.

Trying to find the diff on the package-lock file (especially if the change is actual package version change rather than project version, metadata or some optional settings) is not that straight-forward.

Easiest solution for that I can think of is comparing hashcode of prior package-lock file.

But i reckon using the same check for a package update (via package.json) should be sufficient.

This still has the same behaviour as previous but with just a different install command.

File: src/bin.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
 import postNpmInstall from './index.js';
 
-postNpmInstall();
+const args = process.argv.slice(2);
+setToCi = args.length > 0 && args[0] === 'ci';
```
Comment (jonathantneal) [inline src/bin.js]: 1. Should we include a `const` here?
2. Would it be just as readable to directly access `process.argv`?

```suggestion
const setToCi = process.argv.length > 2 && process.argv[2] === 'ci';
```

**Review by jonathantneal — CHANGES_REQUESTED** [review]
This is awesome. Thank you so much. Would you mind reviewing my comments and seeing if we need to make a couple changes? Seriously, thank you again for taking the time to make this PR.

**Comment by jonyeezs** [comment]
Plus i reckon it is a sound assumption to make as if a predecessor committer would have update/install/remove a package it would have updated both package and lock

**Comment by jonathantneal** [comment]
The reason that I’m not sure checking the hash alone is good enough is because folks often bump package.json without changing any dependencies.

File: src/bin.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import postNpmInstall from './index.js';
 
 const args = process.argv.slice(2);
```
Comment (jonathantneal) [inline src/bin.js]: I think the change below makes this line removable. Thanks for looking at this!

File: src/index.js (hunk 1 above)
Comment (jonyeezs) [reply src/index.js]: definitely

**Review by jonathantneal — APPROVED** [review]

**Comment by jonyeezs** [comment]
@jonathantneal is this going to be publish?

**Pull request merged**